docs/docs/articles/github-actions.md

```diff
@@ -19,8 +20,8 @@ steps:
       testkube create test --name some-test-name --file path_to_file.json
       testkube run test some-test-name -f
 ```
-
-Create and run tests on self-hosted platform. Please mind that it requires establishing connection with the k8s cluster:
+### Testkube Open Source CLI Setup
+Create and run tests on the open source Testkube for AWS provider. Please mind that it requires additional steps to establish connection with the k8s cluster:
```

Comment:
I feel that it would imply that there is some specific version of Testkube for different providers, while it is about Cloud provider for the Kubernetes

Reply:
`...Please mind that it requires additional steps to establish connection with the k8s cluster. While this step will be different for each provider, Testkube Action will stay unchanged.` ?

Reply:
To be honest, I don't know if it's a matter of a single sentence. Maybe we just need to reorganize the documentation, to be more straight-forward - so reading from top to bottom, gives you clear information - then you may either skip a section, or get a knowledge needed to continue, without ever getting back.


As an example, although it's rather badly written:

> ## Configuration for Testkube Open Source
> 
> To use Testkube CLI for Open Source installation, you need to configure your `kubectl` with the Kubernetes cluster, and simply use Testkube Action.
> ```yaml
> steps:
>   # Configure Kubernetes cluster connection
>   # [...] <-- Put here steps for configuring your connection
>
>   # Configure Testkube
>   - uses: kubeshop/setup-testkube@v1
> 
> If you have your Testkube installed in custom namespace, not `testkube`, you need to add a parameter for Testkube Action:
> ```yaml
>   - uses: kubeshop/setup-testkube@v1
>      with:
>        namespace: testkube
> ```
>
> Steps to connect to your Kubernetes cluster differ for each provider. You should check the docs of your Cloud provider on how to connect to the cluster from GitHub Action.
>
> ### Connecting to cluster in AWS EKS
> 
> To connect to AWS EKS, you may use such steps:
> 
> ```yaml
>   - uses: aws-actions/configure-aws-credentials@v4
>     with:
>       aws-access-key-id: ${{ secrets.AwsAccessKeyId }}
>       aws-secret-access-key: ${{ secrets.AwsSecretAccessKey }}
>       aws-region: ${{ secrets.AwsRegion }}
> 
>   - run: |
>       aws eks update-kubeconfig --name ${{ secrets.EksClusterName }} --region ${{ secrets.AwsRegion }}
> ```
> 
> We've used secrets here for vulnerable data, but you may put plain-text information here.
> 
> * The `AwsAccessKeyId`, `AwsSecretAccessKeyId` secrets should contain your AWS IAM keys with proper permissions to connect to EKS cluster.
> * The `AwsRegion` secret should contain AWS region where EKS is
> * Tke `EksClusterName` secret points to the name of EKS cluster you want to connect